test/Login.test.ts

```diff
@@ -0,0 +1,103 @@
+import * as bcryptjs from 'bcryptjs';
+import * as jwt from 'jsonwebtoken';
+import { expect } from 'chai';
+import * as HttpStatus from 'http-status-codes';
+
+import { User } from '../src/entity/User';
+
+export default function loginTest() {
+  describe('Login', function() {
+    const EMAIL = 'email@email.com'
+    const PASSWORD = '1234';
+    const ONE_DAY = 86400;
+
+    const requestLoginMutation = async (email: string, password: string, rememberMe: boolean = false) => {
+      const { request } = this.ctx;
+
+      return await request.post('/').send({ 
+        query: `mutation { Login(email: \"${email}\", password: \"${password}\", rememberMe: ${rememberMe}) { user { id name email birthDate cpf } token }}` });
+    }
+
+    beforeEach(async function() {
+      const { connection } = this.test.ctx;
+
+      const user: User = new User();
+      user.name = 'name';
+      user.email = 'email@email.com';
+      user.birthDate = new Date(1286668800000);
+      user.cpf = '20020020012';
+      user.password = bcryptjs.hashSync('1234');
+
+      await connection.manager.save(user);
+    });
+
+    afterEach(async function() {
+      const { connection } = this.test.ctx;
+      await connection.manager.delete(User, { email: 'email@email.com' });
+    });
+
+    it('should login successfully', async function() {
+      const res = await requestLoginMutation(EMAIL, PASSWORD);
+
+      const { name, email, birthDate, cpf } = res.body.data.Login.user;
+      expect(name).to.be.equals('name');
+      expect(email).to.be.equals('email@email.com');
+      expect(birthDate).to.be.equal('1286668800000');
+      expect(cpf).to.be.equals('20020020012');
+    });
+
+    it('should return all fields', async function() {
+      const res = await requestLoginMutation(EMAIL, PASSWORD);
+
+      const { user, token } = res.body.data.Login;
+      expect(user.id).not.false;
+      expect(user.name).not.false;
```

Comment:
Esse método [false do chai](https://www.chaijs.com/api/bdd/#method_false) compara se o campo é (ou no seu caso, __não é__) estritamente o `false`, como se fosse a comparação `===`. Só que nenhum campo vai ser false, então seu teste sempre dará certo, né?

Reply:
Sim, bom ponto. Eu estava esperando por um valor truthy.

Reply:
Vc pode usar `is.not.empty` pra strings, e `is.gt(0)` pra numeros, por exemplo